src/components/box.js

```diff
@@ -24,6 +24,27 @@
     const isFlex = alignment !== 'none' || valignment !== 'none';
     const opac = transparent ? 0 : 1;
     const [opacity, setOpacity] = useState(opac);
+    const [interactionBackground, setInteractionBackground] = useState('');
+    const [boxStyles, setBoxStyles] = useState({});
```

Comment:
Turns out it wasn't necessary indeed. It was initially because I wanted to prevent from having the `backgroundImage` from being set to `none` when no image was set via the interaction, but setting it directly as the style option and changing the `none` to `auto` fixed this issue.